scripting/lua/LuaScriptingContext.cpp

```diff
@@ -206,7 +205,7 @@ JsonNode LuaContext::callGlobal(const std::string & name, const JsonNode & param
 
 		S.clear();
 
-		return JsonNode();
+		return {};
```

Comment:
I personally like previous version more

Reply:
@dydzio0614 Will look for check which cause this behaviour.

Reply:
@dydzio0614 removed modernization